src/constants.js

```diff
@@ -10,9 +10,11 @@ export const wpRestURL = window.nfdOnboarding.restUrl;
 export const wpRestRoute = 'wp/v2';
 export const onboardingRestRoute = 'newfold-onboarding/v1';
 export const installerRestRoute = 'newfold-installer/v1';
+export const migrationRestRoute = 'newfold-migration/v1';
```

Comment:
Does this route exist in the migration module? If yes, we need to include it as a dependency in Composer.